client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -288,6 +288,6 @@ public T getValue() {
 
   public static final String SPARK_RSS_CONFIG_PREFIX = "spark.";
 
-  public static final boolean RSS_OZONE_DFS_NAMENODE_ODFS_ENABLE_DEFAULT_VALUE = false;
+  public static final boolean RSS_USE_RSS_SHUFFLE_MANAGER_DEFAULT_VALUE = false;
```

Comment:
Why do we modify this?

Reply:
I noticed that I set the name of this parameter wrong.

Reply:
OK